docs/user_guide/metrics.md

```diff
@@ -4,28 +4,30 @@ This doc contains all the metrics that Dragonfly components currently support. N
 
 ## Supernode
 
-- dragonfly_supernode_build_info{version, revision, goversion, arch, os} - build and version information of supernode
-- dragonfly_supernode_http_requests_total{code, handler, method} - total number of http requests
-- dragonfly_supernode_http_request_duration_seconds{code, handler, method} - http request latency in seconds
-- dragonfly_supernode_http_request_size_bytes{code, handler, method} - http request size in bytes
-- dragonfly_supernode_http_response_size_bytes{code, handler, method} - http response size in bytes
+- dragonfly_supernode_build_info{version, revision, goversion, arch, os} - build and version information of supernode.
```

Comment:
@SataQiu Could you please open a new PR to update the metrics in this doc  as a table. THX.

Reply:
OK! @Starnop 
I will do that soon!